src/main/kotlin/eventView/Output.kt

```diff
@@ -0,0 +1,91 @@
+package eventView
+
+import eventViewModel.ValidInput.Companion.benefitPrice
+import eventViewModel.ValidInput.Companion.menu
+import eventViewModel.ValidInput.Companion.result
+import eventViewModel.ValidInput.Companion.totalPrice
+
+class Output {
+    companion object {
+        const val STAR="별"
+        const val TREE="트리"
+        const val SANTA="산타"
+    }
+   private fun printlnOrderMenu(){
+        println(OutputMent.ORDER_MENU.message)
+    }
+    private fun printlnOrderMenuList(){
+        menuMap.forEach{println("${it.key} ${it.value}개")}
+    }
+    fun printlnInPutMenu(){
+        Output().printlnOrderMenu()
+        Output().printlnOrderMenuList()
+    }
+    fun printlnBeforeBenefit(){
+        println(OutputMent.BEFORE_BENEFIT.message)
+    }
+    fun printlnTotalPrice(){
+        println(OutputMent.WON.message.format(totalPrice))
+    }
+    private fun printlnGivewayMenu(){
+        println(OutputMent.GIVEWAY_MENU.message)
+    }
+    fun printlnGiveWayResult():Boolean{
+        printlnGivewayMenu()
+        if(totalPrice>=DiscountPrice.CHAMPAGNE_FREE.price){
+            println(OutputMent.SERVICE_CHANPAGNE.message)
+            return true
+        }
+        println(OutputMent.NO_ONE.message)
+        return false
+    }
+   private fun printlnBenefitMent(){
+        println(OutputMent.BENEFIT_LIST.message)
+    }
+    fun startDays(date:Int,starDays:List<Int>):Boolean{
```

Comment:
[코틀린 컨벤션](https://kotlinlang.org/docs/coding-conventions.html#colon)에서 Always put a space after `:`. 라는 내용이 있기 때문에 이렇게 컨벤션에 맞춰 적어주시면 더 가독성 있는 코드가 될 거 같습니다.
```suggestion
    fun startDays(date: Int, starDays: List<Int>): Boolean {
    }
```